src/main/java/com/nekomaster1000/infernalexp/InfernalExpansion.java

```diff
@@ -111,10 +112,11 @@ private void commonSetup(final FMLCommonSetupEvent event) {
         //Search for all biomes to add to nether and register nether biome provider
         event.enqueueWork(ModNetherBiomeProvider::registerBiomeProvider);
 
-        //Setup and register structures and processors and packets
+        //Setup and register structures, processors, packets and capabilities
```

Comment:
```suggestion
        //Setup and register structures, processors, packets, and capabilities
```
oxford comma :angry: